.github/workflows/config_coverage.yml

```diff
@@ -0,0 +1,150 @@
+          make venv
+          source venv/bin/activate
+          echo "::endgroup::"
+
+          echo "::group::CodeChecker package"
+          make standalone_package
+          deactivate
+          echo "::endgroup::"
+
+          echo "::set-output name=CODECHECKER_PATH::$(readlink -f ./build/CodeChecker/bin)"
+          popd # analyzer
+      - name: "Dump checker list"
+        run: |
+          export PATH="${{ steps.codechecker.outputs.CODECHECKER_PATH }}:$PATH"
+          CodeChecker analyzers --details
+          CodeChecker checkers \
+            --analyzer clangsa clang-tidy \
+            --output rows \
+              > checker_list_normal.txt
+          CodeChecker checkers \
+            --analyzer clangsa clang-tidy \
+            --warnings \
+            --output rows \
+              > checker_list_diagnostics.txt
+      - name: "Perform checker config coverage check (--warnings)"
+        continue-on-error: true
+        run: |
+          set +e # Do not hard exit on an erroring call!
+          .github/workflows/config_label_check.py \
+            "checker_list_diagnostics.txt" \
+            "config/labels/analyzers/clangsa.json" \
+            "config/labels/analyzers/clang-tidy.json" \
+            --existing-filter "clang-diagnostic-" \
+            --new-filter "clang-diagnostic-"
+          EXIT_STATUS=$?
+          echo "Coverage check returned: $EXIT_STATUS."
+          # Explicitly check if the bit for "8" is set in the result,
+          # indicating new checkers without severity set.
+          if [[ $(($EXIT_STATUS & 8)) -eq 8 ]]
+          then
+            echo "::warning title=New unconfigured diagnostics::The checker label config files lack some new diagnostic report (\"warning\") kinds."
+            exit 0
+          elif [[ $EXIT_STATUS -eq 1 || $EXIT_STATUS -eq 2 ]]
+          then
+            # Script execution error.
+            exit $EXIT_STATUS
+          else
+            # We do not wish to fail if only removed checkers are reported.
+            exit 0
+          fi
+      - name: "Perform checker config coverage check"
+        run: |
+          set +e # Do not hard exit on an erroring call!
+          .github/workflows/config_label_check.py \
+            "checker_list_normal.txt" \
+            "config/labels/analyzers/clangsa.json" \
+            "config/labels/analyzers/clang-tidy.json" \
+            --existing-ignore "clang-diagnostic-" \
+            --new-ignore \
+              "clang-diagnostic-" \
+              "alpha." \
+              "apiModeling." \
+              "debug." \
+              "optin.osx." \
+              "osx." \
+              "darwin-" \
+              "objc-"
+          EXIT_STATUS=$?
+          echo "Coverage check returned: $EXIT_STATUS."
+          # Explicitly check if the bit for "8" is set in the result,
+          # indicating new checkers without severity set.
+          if [[ $(($EXIT_STATUS & 8)) -eq 8 ]]
+          then
+            exit 8
```

Comment:
@bruntib This is the execution step which makes the entire job fail if there are new **checkers** that are added, without a label.

Reply:
The job will run scheduled and it will trigger a failure. Once you go into the detailed log of the job, the standard output of the execution will contain the details on which checks are lacking a label:

```
Checkers ADDED to upstream, without labels in CodeChecker:
 + abseil-cleanup-ctad
 + altera-kernel-name-restriction
 + altera-single-work-item-barrier
 + bugprone-shared-ptr-array-mismatch
 + bugprone-stringview-nullptr
 + linuxkernel-must-check-errs
 + misc-misleading-bidirectional
 + misc-misleading-identifier
 + readability-container-contains
 + webkit.UncountedLambdaCapturesChecker
Coverage check returned: 8.
Error: Process completed with exit code 8.
```